src/PkgCoreBundle/objects/TPkgCmsCoreSendToHost.class.php

```diff
@@ -123,23 +123,6 @@ public function getLastResponseHeaderVariable($sVarName)
     {
         if (null === $this->lastResponseHeaderVariables) {
             $this->lastResponseHeaderVariables = array();
-            $rHeader = TPkgCmsStringUtilities::getStringStream($this->getLastResponseHeader());
-            $this->lastResponseCodeRaw = fgets($rHeader);
-
-            $aTmp = explode("\n", $this->getLastResponseHeader());
-            foreach ($aTmp as $sHeaderLine) {
-                $iSplit = strpos($sHeaderLine, ':');
-                $sHeaderVarName = trim(substr($sHeaderLine, 0, $iSplit));
-                $sHeaderVarValue = trim(substr($sHeaderLine, $iSplit));
-                if (isset($this->lastResponseHeaderVariables[$sHeaderVarName])) {
-                    if (!is_array($this->lastResponseHeaderVariables[$sHeaderVarName])) {
-                        $this->lastResponseHeaderVariables[$sHeaderVarName] = array($this->lastResponseHeaderVariables[$sHeaderVarName]);
-                    }
-                    $this->lastResponseHeaderVariables[$sHeaderVarName][] = $sHeaderVarValue;
-                } else {
-                    $this->lastResponseHeaderVariables[$sHeaderVarName] = $sHeaderVarValue;
-                }
-            }
         }
```

Comment:
we should not set an empty array if it wasn't set before. This private attribute has the characteristics of a runtime cache. So setting it to an empty array may signal to other methods that the cache was successfully built, which it wasn't in that case. We could just return null instead and leave the attribute unset.

Reply:
There is no "error" in this case. This is plain initialization.